tests/lsp_features/code_actions.zig

```diff
@@ -378,7 +538,8 @@ fn testAutofixOptions(before: []const u8, after: []const u8, want_zir: bool) !vo
     const handle = ctx.server.document_store.getHandle(uri).?;
 
     var diagnostics: std.ArrayListUnmanaged(types.Diagnostic) = .{};
-    try zls.diagnostics.getAstCheckDiagnostics(ctx.server, ctx.arena.allocator(), handle, &diagnostics);
+    // try zls.diagnostics.getAstCheckDiagnostics(ctx.server, ctx.arena.allocator(), handle, &diagnostics);
+    defer diagnostics.deinit(allocator);
```

Comment:
No deinit is necessary because the diagnostics are allocated with an arena allocator.
```suggestion
    try zls.diagnostics.getAstCheckDiagnostics(ctx.server, ctx.arena.allocator(), handle, &diagnostics);
```
Why the autofix tests are failed should be investigated.